packages/vue/src/Module/createStoreModule.js

```diff
@@ -18,17 +18,17 @@ const createStoreModule = (repository, store) => ({
   },
 
   getters: {
-    resource: state => id => state.resources[id] ? Resource.fromState(
+    resource: state => id => (state.resources[id] ? Resource.fromState(
```

Comment:
You've changed tests and code in this branch, it should be in a seperated branch and PR

Reply:
yup sorry, won't do that next time ;)